src/tomography/rules.mk

```diff
@@ -65,6 +65,24 @@ tomography_MODULES = \
 	$(FC_MODDIR)/tomography_model_iso.$(FC_MODEXT) \
 	$(EMPTY_MACRO)
 
+postprocess_TARGETS = \
```

Comment:
Hi Elliot,

I just want to understand, all the executables in src/tomography are
already included under tomography_TARGETS.  The fact that there is an
additional postprocess target shouldn't be problematic, right?

Thanks,
Ryan

On Mon, Feb 23, 2015 at 2:12 AM, Elliott Sales de Andrade <
notifications@github.com> wrote:

>  In src/tomography/rules.mk
> <https://github.com/geodynamics/specfem3d/pull/429#discussion_r25146776>:
>
> > @@ -65,6 +65,24 @@ tomography_MODULES = \
> >  	$(FC_MODDIR)/tomography_model_iso.$(FC_MODEXT) \
> >  	$(EMPTY_MACRO)
> >
> > +postprocess_TARGETS = \
>
> No, this is wrong. The variables are named after the directory. This
> scheme enables automatic build/clean-only-this-directory if you are in some
> subdirectory. Adding *_TARGETS that are not named after the directory (or
> at least added to that variable) will break that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/429/files#r25146776>.
>